docs/source/User-guide/Spot-text.rst

```diff
@@ -0,0 +1,191 @@
+Spot text
+=========
+
+MapReader now contains two new methods of spotting text on maps:
+
+- ``DPTextDETRRunner`` - This is used to detect text on maps using `DPTextDETR <https://github.com/ymy-k/DPText-DETR/tree/main>`__ and outputs bounding boxes and scores.
+- ``DeepSoloRunner`` - This is used to detect and recognise text on maps using `DeepSolo <https://github.com/ViTAE-Transformer/DeepSolo/tree/main>`__ and outputs bounding boxes, text and scores.
+
+Install dependencies
+--------------------
+
+To run these, you will need to install the required dependencies.
+
+.. note::
+
+    We have our own forks of these repos to enable them to work on CPU!
+
+For DPTextDETR, you need to install the following:
+
+- Detectron2 - main repo available `here <https://github.com/facebookresearch/detectron2>`__
+- DPText-DETR - our fork available `here <https://github.com/rwood-97/DPText-DETR>`__
+
+For DeepSolo, you need to install the following:
+
+- Detectron2 - main repo available `here <https://github.com/facebookresearch/detectron2>`__
+- DeepSolo - our fork available `here <https://github.com/rwood-97/DeepSolo>`__
```

Comment:
For these, I think it would help a lot to list the exact install commands here that you used rather than link to the documentation of each. I just looked at https://github.com/rwood-97/DPText-DETR?tab=readme-ov-file#installation and tried just installing everything into the mapreader conda env I created, so skipping the first 2 lines of the installation instructions - but I get the following error presumably because CUDA isn't installed:
```
fatal error: 'cuda_runtime_api.h' file not found #include <cuda_runtime_api.h
```

I'll see if I can get it to work and comment what I did in the PR - if it proves fiddly perhaps let's co-work on Monday on this

Reply:
oh woops, I will fix that now

Reply:
oh actually the installation instructions were wrong on the README bc they came from the main repo not updated for my fork, should be okay now but we can try on monday